### Pointcloud stuff
cloud_max_interval: 100 # Max 100 seconds per pointcloud
cloud_relative_timestamps: False # NB: Relative timestamps not supported by image reconstruction
min_x_distance: 10

### Use to estimate fov mask ###
baseline_height: 17
lidar_fov: 70.2

#### General LiDAR info ####
lidar_topic: /livox_lidar_node/pointcloud2
lidar_return_count: 2 # Dual return mode
lidar_sample_rate: 100000 # Samples per second
lidar_frame_rate: 10

#### Reconstruction ####
reconstruction_dt: 1
reconstruction_interval: 5
reconstruction_point_size: 1
reconstruction_background_color: [0, 0, 0]

reconstruction_do_overlay: true